lib/idempotency/testing/rspec/helpers.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'idempotency/cache'
+
+class Idempotency
```

Comment:
Could we add documentation for this? I think we don't need to put it in `RSpec` module because it doesn't require RSpec to run